charts/fluid/fluid/templates/controller/jindoruntime_controller.yaml

```diff
@@ -38,6 +38,7 @@ spec:
             - --runtime-node-port-range={{ .Values.runtime.jindo.portRange }}
             - --runtime-workers={{ .Values.runtime.jindo.runtimeWorkers }}
             - --pprof-addr=:6060
+             - --enable-leader-election
```

Comment:
The indentation of `--enable-leader-election` doesn't match the line above.

Reply:
my fault, fixed 😬